### PR TITLE
fix '${BROWSER} resolves to undefined in screenshot pattern' (close #2742)

### LINKED ISSUE #2742: ${BROWSER} resolves to undefined in screenshot pattern
### Are you requesting a feature or reporting a bug?

bug

### What is the current behavior?
when using ${BROWSER} in screenshot pattern it resolves to undefined

### What is the expected behavior?
should resolve to the browser name

### How would you reproduce the current behavior (if this is a bug)?
use any pattern with ${BROWSER}

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```js

```

### Specify your

* operating system: Windows 10
* testcafe version: 0.21.0
* node.js version: 0.8.3

### PR DESCRIPTION
**Pull request by miherlosev**
@AndreyBelym 

Review only two files:
* `test/server/path-pattern-test.js`
*  `src/screenshots/path-pattern.js`

### CODE REVIEW

File: test/server/path-pattern-test.js [hunk 1]
```diff
@@ -27,46 +33,39 @@ describe('Screenshot path pattern', () => {
...
+            'Windows',
+            '8.1.0.0'
         ].join('#') + '.png';
 
         const pathPattern = createPathPattern(pattern, data);
 
         const path = pathPattern.getPath(false);
 
-        expect(path).eql(expectedParsedPattern);
+        expect(path).contains(expectedParsedPattern);
```
Comment (AndreyBelym) [inline test/server/path-pattern-test.js]: Is it an intentional change?

Comment (miherlosev) [reply test/server/path-pattern-test.js]: Good catch. It's from development.

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Pull request merged**